.coveragerc

```diff
@@ -0,0 +1,19 @@
+
+[run]
+branch = True
+omit =
+        */test/*
+        */usr/local/lib*
+        */__init__.py
+        */*/chalicelib/common/utilities.py
+        */app.py
+        */database_populator.py
```

Comment:
suspect this would need to be **/*database_populator.py too.

Reply:
agreed..have changed it